pkg/client/event.go

```diff
@@ -19,14 +19,14 @@ type BulkPushOpFunc func(*eventcontracts.BulkPushEventRequest) error
 type EventClient interface {
 	Push(ctx context.Context, eventKey string, payload interface{}, options ...PushOpFunc) error
 
-	BulkPush(ctx context.Context, payloads []EventWithMetadata, options ...BulkPushOpFunc) error
+	BulkPush(ctx context.Context, payloads []EventWithAdditionalMetadata, options ...BulkPushOpFunc) error
```

Comment:
why did this make its way into this PR?

Reply:
I think it's just something weird with the merge, I'm pretty sure this is all in main already